source/common/protobuf/utility.cc

```diff
@@ -68,6 +78,11 @@ void MessageUtil::loadFromFile(const std::string& path, Protobuf::Message& messa
   if (StringUtil::endsWith(path, ".pb")) {
     // Attempt to parse the binary format.
     if (message.ParseFromString(contents)) {
+      if (!MessageUtil::allow_unknown_fields &&
```

Comment:
Can you factor this out to a common utility?

Reply:
Done